src/site/twitch.tv/modules/chat/ChatPubSub.vue

```diff
@@ -103,6 +107,25 @@ async function onModerationAction(msg: PubSubMessageData.ModAction) {
 	}
 }
 
+async function onChatRichEmbed(msg: PubSubMessageData.ChatRichEmbed) {
+	const ctx = msg.twitch_metadata;
+
+	if (!ctx) return;
+
+	// Find the message
+	const message = messages.messageById(msg.message_id);
```

Comment:
This is most likely why, use `messages.awaitMessage` instead.

Reply:
I am using `messages.messageById` because the chat message arrives before the pubsub message does (on the sender), but this is the opposite (on the receiver).